squad/core/migrations/0096_build_keep_data.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.16 on 2018-10-18 12:13
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0095_project_data_retention_days'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='build',
+            name='keep_data',
+            field=models.NullBooleanField(help_text='Keep this build data even after the project data retention period has passed'),
```

Comment:
Shouldn't we rather use BooleanField with default set to False?
https://docs.djangoproject.com/en/2.1/ref/models/fields/#booleanfield

Reply:
fair enough. Changed